tests/neg/scala-uri.scala

```diff
@@ -0,0 +1,31 @@
+//> using options -source:3.6
```

Comment:
Backported version required -source:3.6 which is not needed in main

Reply:
main has the baseVersion set to 3.6 already, so we get the `CompareScheme.New` by default. 
While in 3.5 we maintain the intermediate comparison scheme, in which the test case used to be un-ambiguous, so I think this is as expected.